kvmagent/kvmagent/plugins/virtiofs_plugin.py

```
@@ -328,9 +355,16 @@ def mount_virtiofs_in_vm(domain, tag, mount_path):
         return True, None
 
     except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)